src/style.css

```diff
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: bold;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.input {
+  box-sizing: border-box;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-weight: bold;
+  color: #3d4e61;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  outline: none;
+}
+
+.input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.big-input {
+  height: 70px;
+  font-size: 16px;
+  padding-left: 62px;
+}
```

Comment:
move this styles to the common selector and override them for small input as you did
get rid of this selector
same for icons